tuxemon/resources/db/locale/es_ES.json

```diff
@@ -473,5 +473,7 @@
     "okaythen": "¡De acuerdo! ¡Dame un segundo y curaré a tus Tuxemon!",
     "okaythen2": "¡Bueno, parece que tus Tuxemon están llenos de energía! ¡Aquí los tienes!",
 
-    "input_name": "¿Cuál es tu nombre?"
+    "input_name": "¿Cuál es tu nombre?",
+    "dont_care": "No importa.",
+    "default_names": ["Cruz","Loreto","Lupe","Chus","Cande","Ariel","Ale","Manu","Zorion","Reyes","Odalis","Nazaret"]
```

Comment:
Those are very unusual names (except Manu). 

- I have never met (or heard about) any Spanish person called "Cruz" (this is a surname), "Ariel", "Ale", "Zorion", "Reyes" (maybe in Mexico?), "Odalis" or "Nazaret".

- Some of the remaining ones are diminutives: "Lupe" is "Guadalupe" (uncommon nowadays), "Chus" is "María Jesús" (in this case I think "Chus" is better), "Cande" is "Candela"(also uncommon) and "Manu" is "Manuel".

- "Loreto" exists, but is uncommon too.

Reply:
Also, some names are male names while other are female ones.

Reply:
Hello @vnmabus! Thank you for giving us feedback. @silvercam15 and I chose unusual names for each locale because our intention was to keep the default names gender-neutral. [Behind the Name](https://www.behindthename.com/) indicated that the names you mentioned above were all names that could be applied to both males and females, which is why we chose them. If they are too unusual, we would be more than happy to change the names to more common names. Thanks again!

Reply:
Gender-neutral names and Spanish do not mix well. Even the diminutives that, in principle, could come from a male or a female name, such as "Manu" meaning either "Manuel" or "Manuela", often are used mostly for one sex (in this case, "Manuel", which is a more common name than "Manuela"). If you want unisex identification, maybe you should assign a surname instead.

Reply:
@vnmabus you seem to have a good idea of spanish names that would work.  Right now, there is not an easy way to assign or determine gender of the monsters.  Do you mind if I merge this, and if you care to, you could assign the surnames to the spanish locale?

Reply:
Well, the name is not for the monsters but for the main player... I always though he was male.

Reply:
The default main character at the moment is male - but there are designs for a female main character to be added at some point.